src/java.desktop/windows/native/libawt/windows/awt_Window.cpp

```diff
@@ -1406,25 +1406,17 @@ BOOL AwtWindow::UpdateInsets(jobject insets)
     } else {
         m_insets.top = -1;
     }
+
     if (m_insets.left < 0 || m_insets.top < 0 ||
-        m_insets.right < 0 || m_insets.bottom < 0)
-    {
+        m_insets.right < 0 || m_insets.bottom < 0) {
```

Comment:
No strong opinion here, yet I'd rather leave these lines unchanged. Having the opening brace on its own line avoids the confusion between the condition and the code in the body; on the other hand, it starts with a comment which visually separates the condition from the code inside the if-block.